js/package-lock.json

```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "js",
```

Comment:
don't need this change

Reply:
Done. This was introduced after running "npm run format;npm run lint" with rebase. Revert this change.